app/components/Welcome.js

```diff
@@ -19,25 +32,56 @@ export default class Welcome extends Component {
     setNotFirstTimeFlag()
   }
 
+  selectedLanguage (i) {
```

Comment:
Not needed move it to onLanguageChange and use 
`this.setState({ language: event.target.value })`

Reply:
I also tried doing it in one method, didn't worked correctly. I'll try again